src/validationinterface.cpp

```diff
@@ -291,6 +301,56 @@ void ThreadNotifyWallets(CBlockIndex *pindexLastTip)
         // for example to add new incoming viewing keys.
         auto batchScanners = GetMainSignals().GetBatchScanner();
 
+        // Closure that returns true if batchScanners is using less memory than
+        // the desired limit.
+        auto belowBatchMemoryLimit = [&]() {
+            return RecursiveDynamicUsage(batchScanners) < DEFAULT_BATCHSCANNERMEMLIMIT;
```

Comment:
Isn't `RecursiveDynamicUsage(batchScanners)` in bytes and `DEFAULT_BATCHSCANNERMEMLIMIT` in MiB?

Reply:
Ah yes, this is a refactor bug from when I removed the config option. I'll fix